framework/include/fwk_interrupt.h

```diff
@@ -56,6 +57,14 @@
  */
 #define FWK_INTERRUPT_EXCEPTION (UINT_MAX - 2)
 
+/*!
+ * \brief Initialize interrupt management in the framework.
+ *
+ * \retval FWK_SUCCESS Operation succeeded.
+ * \retval FWK_E_PARAM One or more parameters were invalid.
+ */
+int fwk_interrupt_init(const struct fwk_arch_interrupt_driver *_driver);
```

Comment:
`_driver` -> `driver` - the underscore exists to differentiate the static global in the `.c` file.

Reply:
ok, so i'll change fwk_interrupt.c line 23, ok?
```patch
 static const struct fwk_arch_interrupt_driver *driver;
 static unsigned int critical_section_nest_level;
 
-int fwk_interrupt_init(const struct fwk_arch_interrupt_driver *_driver)
+int fwk_interrupt_init(const struct fwk_arch_interrupt_driver *driver)
 {
     /* Validate driver by checking that all function pointers are non-null */
     if (_driver == NULL)
```

Reply:
discard my last comment :)